cgi-bin/LJ/Web.pm

```diff
@@ -886,7 +886,7 @@ sub create_qr_div {
     }
 
     return DW::Template->template_string( 'journal/quickreply.tt', {
-        form_url                => LJ::create_url( '/talkpost_do', host => $LJ::DOMAIN_WEB ),
```

Comment:
This is for the use-case when we’re passing in the host option — and
since we can’t guarantee that the new host supports ssl, we want to pass
in the ssl flag.

In most other cases, create_url is smart enough to do things as you
described.